proto/primitive/session_context.proto

```diff
@@ -25,6 +30,6 @@ message SessionContext {
 	string tenancy_name = 3;
 	string user_name = 4;
 	.holms.types.iam.StaffMemberIndicator user = 5;
-	repeated Capability capabilities = 6;
+	repeated UserEntitlements user_entitlements = 6;
```

Comment:
To summarize, there are basically two ways forward here:

(1) We treat tokens as non-opaque. In this case, (1) keep jwtoken in this shared library, (2) remove the entire field, and (3) write, and make usable, the code to tell what entitlements are conveyed by the token. If we go this route, we should also drop the expiration time, since it also duplicates what's in the token.

(2) We keep tokens as opaque. We get rid of the ability to read and write tokens from this library, repeat all salient data from the token including expiry that we want to the client to have, and treat the token as a raw `string` that the client can't mess with.

Open to suggestions on which is better -- let me know your thoughts.